Slider/SliderBehaviorDecorator.js

```diff
@@ -132,7 +132,7 @@ const SliderBehaviorDecorator = hoc(defaultConfig, (config, Wrapped) => {
 
 		handleDragStart () {
 			// on platforms with a touchscreen, we want to focus slider when dragging begins
-			if (platform.touchscreen) {
+			if (platform.touchScreen) {
```

Comment:
Drag doesn't work on touch mode

Reply:
Please check the building environment as I successfully build and make it work on several environments.